MapleServer2/Types/Currency.cs

```diff
@@ -0,0 +1,86 @@
+﻿using MapleServer2.Enums;
+using MapleServer2.Packets;
+
+namespace MapleServer2.Types
+{
+    public class Currency
+    {
+
+        public long Amount { get; private set; }
+        private readonly CurrencyType Type;
+        private readonly Player Player;
+
+        public Currency(CurrencyType type, long input, Player player)
+        {
```

Comment:
Can the order of the fields at the top, the parameters, and the initialization inside the constructor all be the same?